src/targets.js

```diff
@@ -20,6 +20,7 @@ const buildVersions = {
   },
   linux: {
     arm64: '>= 1.8.0',
+    ia32: '<19.0.0-beta.1',
```

Comment:
Was this an alpha/beta cutoff change or an 18/19 cutoff change?

```suggestion
    ia32: '<19.0.0-alpha.1',
```

Reply:
It wasn't the latter. From the PR summary:

> Compare the assets of [19.0.0-alpha.5](https://github.com/electron/electron/releases/tag/v19.0.0-alpha.5) with [19.0.0-beta.1](https://github.com/electron/electron/releases/tag/v19.0.0-beta.1).

I'd have to look at either PRs or Slack context to see what the issue was (if I had to guess, ia32 didn't actually work in alpha, but :shrug:)

Reply:
From some internal maintainer discussion, we technically we publicly announced that we were going to drop support for Linux in Electron 4 but then never removed it from CI: https://www.electronjs.org/blog/linux-32bit-support

Reply:
I think I'm good with either option, though.